core/sql/common/ComMisc.cpp

```diff
@@ -228,3 +228,17 @@ NAString ComConvertTrafNameToNativeName(
   return convertedName;
 }
 
+NABoolean ComTrafReservedColName(
+     const NAString &colName)
```

Comment:
I take it that by the time we get here, colName has already been case-shifted to upper case?